test/gurka/gurka.h

```diff
@@ -231,8 +234,15 @@ std::string build_valhalla_match_request(const map& map,
   rapidjson::StringBuffer sb;
   rapidjson::Writer<rapidjson::StringBuffer> writer(sb);
   doc.Accept(writer);
-  std::cout << sb.GetString() << std::endl;
-  return sb.GetString();
+  std::string request = sb.GetString();
+  // inject more options directly
+  if (!trace_options.empty()) {
```

Comment:
we needed the ability to send more options to the trace request so i've spliced it in here. since its a free form json object i just used a string but i technically could have added a rapidjson doc. i feel like for tests its way less of a hassle this way